packages/grid-html-export/src/components/StandardTableHtmlCopyToClipboardButton.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import { useCallback, useState } from "react";
+import { StandardTableProps } from "@stenajs-webui/grid";
+import { FlatButton, FlatButtonProps } from "@stenajs-webui/elements";
+import * as clipboard from "clipboard-polyfill";
+import { CustomCellFormatters } from "../transformers/ConfigTransformer";
+import { renderHtmlForStandardTable } from "../util/HtmlRenderer";
+
+interface StandardTableHtmlExportButtonProps<
```

Comment:
`StandardTableHtmlExportButton` should probably be renamed to `StandardTableCopyToClipboardButton` to best indicate functionality.